Themis/Views/Assessment/UML/UMLView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ImageView.swift
+//  Themis
+//
+//  Created by Evan Christopher on 10.12.22.
+//
+
+import Foundation
+import SwiftUI
+
+struct UMLView: View {
+    @Environment(\.colorScheme) var colorScheme
+    @EnvironmentObject var umlVM: UMLViewModel
+    @GestureState var draggingOffset: CGSize = .zero
+
+    var body: some View {
+        ZStack {
```

Comment:
At first I thought this `ZStack` might not be needed as well, but then the X button on the top right is not completely at the top, so it's fine 👍 

Edit: I see, it's also for the popup background, clever!